doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>not that important, but I would try to be consistent with the docstring conventions of writing modules, variables etc.</p><p> </p><p>From what I remember, the <code>y</code> argument is optional for classification. Shall we be explicit about it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>